ui/packages/app/src/project_setting/form/ProjectForm.js

```diff
@@ -57,31 +57,41 @@ const ProjectForm = () => {
     setName(newValue);
   };
 
+  const [streamError, setStreamError] = useState("");
+  const [isValidStream, setIsValidStream] = useState(false);
+  const onStreamChange = selectedStream => {
+    if (selectedStream !== project.stream) {
+      let isValid = isDNS1123Label(selectedStream);
```

Comment:
Just realised yup validations are not integrated to MLP app. Is this tech debt? Or did we choose not to do it for some reason?

Reply:
It's a debt. The entire `ui/packages/app` is a debt to some extent.